packages/api-v4/src/vpcs/types.ts

```diff
@@ -52,31 +53,37 @@ export interface ModifySubnetPayload {
 }
 
 export interface SubnetLinodeInterfaceData {
-  id: number;
   active: boolean;
-  config_id: number | null;
+  config_id: null | number;
+  id: number;
 }
 
 export interface SubnetAssignedLinodeData {
   id: number;
   interfaces: SubnetLinodeInterfaceData[];
 }
 
+export interface SubnetAssignedNodeBalancerData {
+  id: number;
+  ipv4_range: string;
+}
+
 export interface VPCIP {
   active: boolean;
-  address: string | null;
-  address_range: string | null;
-  ipv6_range: string | null;
-  ipv6_is_public: boolean | null;
+  address: null | string;
+  address_range: null | string;
+  config_id: null | number;
+  gateway: null | string;
+  interface_id: number;
   ipv6_addresses: {
     slaac_address: string;
   }[];
-  config_id: number | null;
-  gateway: string | null;
-  interface_id: number;
+  ipv6_is_public: boolean | null;
+  ipv6_range: null | string;
   linode_id: number;
```

Comment:
I believe this should be: 
```suggestion
  linode_id: null | number;
```